tests/Integration/Behaviour/Features/Scenario/Order/add_cart_rule_to_order.feature

```diff
@@ -43,6 +43,14 @@ Feature: Add discounts to order from Back Office (BO)
       | taxes         | $1.85    |
       | total         | $32.65   |
 
+  Scenario: Add amount type discount to order which has no invoices with HTML characters
```

Comment:
```suggestion
  Scenario: Add a discount to order (no invoices) whose name contains HTML characters
```
Maybe a little bit easier to understand 📚 👩‍🎨

Reply:
Fixed